lxd/daemon.go

```diff
@@ -25,6 +25,7 @@ import (
 	liblxc "github.com/lxc/go-lxc"
 	"golang.org/x/sys/unix"
 
+	lxd "github.com/canonical/lxd/client"
```

Comment:
no

Reply:
Ah I missed this. It's my IDE not playing ball when doing automatic imports :facepalm:

Reply:
Fixed and added a commit for the other occurrence.